bin/protoc-gen-ruby_gapic

```diff
@@ -0,0 +1,93 @@
+#!/usr/bin/env ruby
```

Comment:
I would put most of the logic here into a separate class, and just have the binary itself call that class. This is so you can manage and refactor the code (e.g. break out methods) more easily.

Reply:
Done!